src/main/kotlin/com/mjucow/eatda/domain/notice/service/command/NoticeCommandService.kt

```diff
@@ -0,0 +1,35 @@
+package com.mjucow.eatda.domain.notice.service.command
+
+import com.mjucow.eatda.domain.notice.entity.Notice
+import com.mjucow.eatda.domain.notice.service.command.dto.CreateNoticeCommand
+import com.mjucow.eatda.domain.notice.service.command.dto.UpdateNoticeCommand
+import com.mjucow.eatda.persistence.notice.NoticeRepository
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+@Transactional
+class NoticeCommandService(
+    private val repository: NoticeRepository,
+) {
+    fun create(request: CreateNoticeCommand): Long {
+        return repository.save(Notice(request.title, request.content)).id
+    }
+
+    fun update(noticeId: Long, command: UpdateNoticeCommand) {
+        val (newTitle, newContent) = command
+        val notice = repository.findByIdOrNull(noticeId)?.apply {
+            title = newTitle
+            content = newContent
+        } ?: throw IllegalArgumentException()
+
+        repository.save(notice)
```

Comment:
[nit] 이미 업데이트 되어진 것 변수에 나타내기?!
```suggestion
        val updatedNotice = repository.findByIdOrNull(noticeId)?.apply {
            title = newTitle
            content = newContent
        } ?: throw IllegalArgumentException()

        repository.save(updatedNotice)
```